app/app.scss

```diff
@@ -42,3 +42,8 @@
 #doorbell-background {
     opacity: 0;
 }
+
+.smartcharts-mobile .cq-chart-controls .ciq-menu .cq-menu-overlay
+{
+    height: calc(var(--view-height) - 39px) !important;
```

Comment:
@MahboobehMohammadi @brucebinary Css variables are still [new](https://caniuse.com/#feat=css-variables), only the latest version of browsers are supporting them.